develop/networks.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Networks
+description: Explore the Polkadot ecosystem networks and learn the unique purposes of each, tailored for blockchain innovation, testing, and enterprise-grade solutions.
+template: root-subdirectory-page.html
+---
+
+# Networks
+
+## Introduction
+
+The Polkadot ecosystem consists of multiple networks designed to support different stages of blockchain development, from main networks to test networks. Each network serves a unique purpose, providing developers with flexible environments for building, testing, and deploying blockchain applications.
+
+## Main Networks
+
+### Polkadot
+
+- **Description** - primary production blockchain network for high-stakes, enterprise-grade applications
+- **RPC URL** - `wss://polkadot-rpc.dwellir.com`
+- **Currency symbol** - `DOT`
+- **Currency decimals** - 10
+- **Block explorer URL** - [`https://polkadot.subscan.io/`](https://polkadot.subscan.io/){target=\_blank}
+- **Network Characteristics:**
+    - Highest security and stability
+    - Real economic incentives
+    - Production-ready environment
+    - Advanced governance mechanisms
+
+### Kusama
+
+- **Description** - experimental network that receives features before Polkadot
+- **RPC URL** - `wss://kusama-rpc.dwellir.com`
+- **Currency symbol** - `KSM`
+- **Currency decimals** - 12
+- **Block explorer URL** - [`https://kusama.subscan.io/`](`https://kusama.subscan.io/){target=\_blank}
+- **Network Characteristics:**
+    - Fast-moving "canary" environment
+    - Real economic incentives
+    - Rapid governance cycles
+    - Testing ground for Polkadot features
+    - Higher risk tolerance
+
+## Test Networks
+
+### Westend
+
+- **Description** - test network that mirrors Polkadot's functionality for protocol development
+- **RPC URL** - `wss://westend-rpc.dwellir.com`
+- **Currency symbol** - `WND`
+- **Currency decimals** - 12
+- **Block explorer URL** - [`https://westend.subscan.io/`](https://westend.subscan.io/){target=\_blank}
+- **Faucet** - [`https://faucet.polkadot.io/westend`](https://faucet.polkadot.io/westend){target=\_blank}
```

Comment:
Up to you here, but maybe it'd be helpful to add a brief sentence before talking about TestNets to explain why these networks use or need a faucet, and also that they don't have a "Real economic incentives/value"